app/pages/EoriDetailsTemporaryPage.scala

```diff
@@ -19,10 +19,10 @@ package pages
 import models.ContactAddress
 import play.api.libs.json.JsPath
 
-case object ImporterAddressTemporaryPage extends QuestionPage[ContactAddress] {
+case object EoriDetailsTemporaryPage extends QuestionPage[ContactAddress] {
```

Comment:
Please rename to KnownEoriDetails because it's no longer temporary info. The name will be used in the submission. Also the type of data stored should be the `EoriDetails`
```suggestion
case object KnownEoriDetails extends QuestionPage[EoriDetails] {
```